Bottle v0.12.13 server starting up (using AutoServer())...
Listening on http://0.0.0.0:80/
Hit Ctrl-C to quit.

[E 180116 23:58:27 robot:405] Catch an exception
    Traceback (most recent call last):
      File "/usr/local/lib/python3.4/dist-packages/werobot/robot.py", line 399, in get_reply
        reply = handler(*args)
      File "auto_return_articles.py", line 40, in hello
        robot.client.send_article_message(message.source,articles)
      File "/usr/local/lib/python3.4/dist-packages/werobot/client.py", line 889, in send_article_message
        "articles": formatted_articles
      File "/usr/local/lib/python3.4/dist-packages/werobot/client.py", line 74, in post
        **kwargs
      File "/usr/local/lib/python3.4/dist-packages/werobot/client.py", line 60, in request
        if check_error(json):
      File "/usr/local/lib/python3.4/dist-packages/werobot/client.py", line 21, in check_error
        raise ClientException("{}: {}".format(json["errcode"], json["errmsg"]))
    werobot.client.ClientException: 40003: invalid openid hint: [Qmc7fa0307ge21]
[W 180116 23:58:27 robot:419] No handler responded message <werobot.messages.messages.TextMessage object at 0xb6b5764c>
182.254.86.149 - - [16/Jan/2018 23:58:27] "POST /?signature=fc0dd1e90f8e80e07c74fd81a327bf7bc98c616d&timestamp=1516118306&nonce=1263752676&openid=o2NddxHhZloQV55azmx8zVXv9mAQ HTTP/1.1" 200 0
《金刚》
